oomph-vscode/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+*.vsix
```

Comment:
Can we put these to oomph's main `.gitignore`

Reply:
Sure thing! You might want to prefix them with `oomph-vscode/`, maybe?

Reply:
Where do vsix files get generated` In `oomph-vscode/` only or also inside `oomph-vscode/some_subdirectory` too?

Reply:
I think (reading from [here](https://code.visualstudio.com/api/working-with-extensions/publishing-extension)) it's in the same directory.